verification/curator-service/api/src/index.ts

```diff
@@ -130,6 +131,7 @@ const awsEventsClient = new AwsEventsClient(
     awsLambdaClient,
     env.SERVICE_ENV,
 );
+const s3Client = new AWS.S3({ region: 'us-east-1', signatureVersion: 'v4' });
```

Comment:
Is it right, or should it be us-east-2?

Reply:
us-east-1 is correct! many of the s3 buckets are on 1